code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/LDAPIdentityProvider.java

```diff
@@ -211,6 +219,28 @@ protected Identity attributesToIdentity(LdapName dn){
         }
     }
 
+    private String removeSpacesAroundRDN(String dn) {
+
+        if (dn == null || StringUtils.isEmpty(dn)) {
```

Comment:
I believe stringUtils.isEmpty returns true for null string. So the first check condition can be skipped